src/rtl_433.c

```diff
@@ -1032,7 +1032,7 @@ int main(int argc, char **argv) {
                 }
                 break;
             case 'U':
-#if !defined(__MINGW32__)
+#if !defined(_WIN32)
```

Comment:
Although  there is no `setenv()` on MinGW/MSVC, one can use `putenv()`:
```diff
--- a/src/rtl_433.c 2017-12-06 22:08:24
+++ b/src/rtl_433.c 2017-12-07 13:38:45
@@ -1032,9 +1032,13 @@
                 }
                 break;
             case 'U':
-#if !defined(__MINGW32__)
+#ifdef _WIN32
+                putenv ("TZ=UTC+0");
+                _tzset();
+#else
                 utc_mode = setenv("TZ", "UTC", 1);
-                if(utc_mode != 0) fprintf(stderr, "Unable to set TZ to UTC; error code: %d\n", utc_mode);
+                if(utc_mode != 0)
+                  fprintf(stderr, "Unable to set TZ to UTC; error code: %d\n", utc_mode);
 #endif
                 break;
             case 'W':
```

Ref: https://docs.microsoft.com/en-gb/cpp/c-runtime-library/reference/tzset

Reply:
thanks! updated.